core/amber/src/main/scala/edu/uci/ics/texera/web/TexeraWebApplication.scala

```diff
@@ -102,7 +101,19 @@ object TexeraWebApplication {
         .toString
     )
 
-    OPversion.refreshVersion()
+    val operatorPath = Map(
+      "Regex" -> "core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/regex",
```

Comment:
Please declare “operator_path” variable to include the path until operator then reuse the variable everytime and append each operator’s foldername

Reply:
use another approach